### ISSUE TITLE
pip install can't find cbors

### ISSUE THREAD

**Issue by ethercrow** [mjkoo/cbors#2]
```
> pip3 install cbors
ERROR: Could not find a version that satisfies the requirement cbors (from versions: none)
ERROR: No matching distribution found for cbors
```

**Comment by mjkoo** [comment]
Thanks for report, I can reproduce and will look into this.

**Comment by mjkoo** [comment]
I believe this should be resolved as of 0.1.4 (pushed yesterday), let me know if there are still issues and I will reopen. Thanks!
